src/users/user-router.js

```diff
@@ -1,11 +1,21 @@
-var express = require('express');
-var userManager = require('./user-manager.js');
-var userRouter = express.Router();
+const express = require('express');
+const userRouter = express.Router();
 
-userRouter.post('/', (req, res) => {
-  var response = userManager.createUser(req.body);
+const userManager = require('./user-manager.js');
 
-  res.status(201).json(response.object);
+// Create User.
+userRouter.post('/', (req, res) => {
+  userManager.createUser(req.body, (err, id) => {
+    if (err) {
+      res.status(500).json(err).end();
+    } else if (id) {
+      // A student was created.
+      res.status(201).json({ studentId: id }).end();
+    } else {
+      // An admin was created.
+      res.status(201).end();
```

Comment:
```id``` should be passed to this callback regardless of role, because it is the id of the document (which can be used with ```findById()```).